src/Oscoin/Crypto/Hash/Mock.hs

```diff
@@ -49,7 +49,9 @@ instance HasHashing MockCrypto where
     newtype Hash MockCrypto = FnvHash { fromMockHash :: FnvHash64 }
         deriving (Eq, Ord)
 
-    fromByteArray = FnvHash . hashByteArray
+    -- | Hash anything which is an instance of 'ByteArray' into a 'Word64' using
+    -- the xxhash.
```

Comment:
Note to myself: this comment is now stale, as we _used_ to use `xxhash` as non-cryptographic hashing function, but now we are using `FNV`, and we should probably amend it 😉